src/Main.js

```diff
@@ -0,0 +1,142 @@
+import React from 'react';
+import ReactDOM from 'react-dom';
+import "./Main.css"
+import "../node_modules/bootstrap/dist/css/bootstrap.min.css"
+import "../node_modules/bootstrap/dist/js/bootstrap.min.js"
+import "../node_modules/jquery/src/jquery.js"
+import "../node_modules/popper.js/dist/popper.min.js"
+
+
+class NavbarOption extends React.Component {
+  render() {
+    return <a href={this.props.link}>{this.props.text}</a>;
+  }
+}
+
+class Navbar extends React.Component {
+  renderNavbarOption(option, link) {
+    return <NavbarOption text={option} link={link} />;
+  }
+
+  render() {
+    return (
+   <nav class="navbar navbar-expand-sm bg-dark navbar-dark">
```

Comment:
I'm not sure the navbar is rendering correctly when the window is reduced to a mobile screen size. All the options should collapsed into an overflow menu - check out the Bootstrap implementation:

https://getbootstrap.com/docs/4.0/components/navbar/